docs/docs/04-standard-library/util/api-reference.md

```diff
@@ -256,6 +256,13 @@ util.ulid(options?: UlidOptions);
 
 Generates universally unique lexicographically sortable identifier.
 
+This works with a precision of 1ms, which might not be sufficient for some cases where the strict order of
+creation is a desired property - e.g. batch creation of resources. In these cases, the generated IDs will
+be still unique (can generate 1.21e+24 unique ULIDs per millisecond) but might not be strictly reflecting the order of creation.
+To mitigate this, you could sleep for 1ms between creation of resources. However, in distributed
+systems where the strict order of creation is a requirement, this is not sufficient
+and you should consider using a different ID generation strategy.
```

Comment:
Out of curiosity are you still using `ulid` for your use case or did you switch to another library/solution? (just trying to gauge usage)

Reply:
not using it right now, but would personally use this for most of the cases where I want to generate ids for records.